src/tools/spark-sstfile-generator/README.md

```diff
@@ -1,9 +1,11 @@
-Generate sst files from hive tables datasource, guided by a mapping file, which maps hive tables to vertexes and edges.
-Multiple vertexes or edges may map to a single hive table, where a partition column will be used to distinguish different
-vertex or edge.  
-The hive tables may be periodically be regenerated by upstream system to reflect the latest data in so far, and may be
-partitioned by a time column to indicate the time when data are generated.  
-*$HADOOP_HOME* env need to be set for running this job.  
+Guided by a mapping file, sst files are generated from hive tables datasource. A mapping file maps hive tables to vertexes and edges.
```

Comment:
sst is the abbreviation of `Sorted String Table` . Maybe using `SST` will be better ?

Reply:
sst is acceptable